.circleci/config.yml

```diff
@@ -1,8 +1,6 @@
-version: 2
-jobs:
-  build:
-    parallelism: 1
-    working_directory: ~/app-prototype
+version: 2.1
+executors:
```

Comment:
FYI, I think the main 2.1 feature in use is defining this shared executor to DRY up the environment declaration.